cloud/aws-ec2/README.md

```diff
@@ -0,0 +1,45 @@
+# Deploying to AWS EC2 using Terraform
+This example launches an EC2 instance, deploys the [SecretHub Demo App](https://secrethub.io/docs/start/getting-started/#consume) on it and provisions it with the required secrets using the [SecretHub AWS Identity Provider](https://secrethub.io/docs/reference/aws/). 
+
+## Prerequisites
+1. [Terraform](https://www.terraform.io/downloads.html) installed
+2. [SecretHub](https://secrethub.io/docs/start/getting-started/#install) installed
+3. Correctly configured [AWS Credentials](https://www.terraform.io/docs/providers/aws/index.html#authentication)
+
+## Running the example
+
+Init the SecretHub demo repo with example values
+```
+secrethub demo init
+```
+
+Initialize a new Terraform working directory
+```
+terraform init
+```
+
+Launch the EC2 instance by running
+```
+terraform apply
+```
+and providing the requested values.
```

Comment:
I think we can be brief and to the point here, it's not a guide:

````suggestion
To launch an example EC2 instance (`t2.nano`), run:
```
terraform init
terraform apply
```
````